ui/src/plugins/Dialog.json

```diff
@@ -13,7 +13,7 @@
       "params": {
         "opts": {
           "desc": "Dialog options",
-          "tsType": "QDialogOptions",
+          "tsType": "DialogOptions",
```

Comment:
Care about this: it was the only one specified because it was the only one used.
Changing the `tsType` here you could have broken something (not sure tho, I only found one reference for this in the repo).

Does the generated `index.d.ts` file contain any errors?

Reply:
I see no errors in the file (vetur, eslint, etc) unless I am doing it wrong.

Reply:
There is now 8494 lines in index.d.ts

Reply:
In regards to QDialogOptions => DialogOptions
Because QDialog and Dialog (global) are two separate entities, I felt it should not have the `Q`
The acess point is `this.$q.dialog` and the other is `<q-dialog ...`